src/components/ListView.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../styles/listView.css';
+
+const ListView = () => {
+  return (
+    <div className="list-View">
+      <ul>
+        <li className="list">
+          <Link to="/list-view">List View</Link>
+        </li>
+        <li>
+          <Link to="/add-view">Add View</Link>
```

Comment:
This is a good first stab at this, and I'm seeing a bit of code that is reused in all of the components in order to keep the links on the page as you're navigating to different components. Fortunately, it looks like `react-router-dom` has a way to do this, so that you can render out only the bits that the component is responsible for.

I think the pattern we should try to follow is what is shown in this example: [1st Example: Basic Routing](https://reactrouter.com/web/guides/quick-start/1st-example-basic-routing). It allows you to declare these links in the `app.js` component alongside the route switches and will greatly reduce the amount of code reuse here.

Check out the [Navigation (or Route Changers)](https://reactrouter.com/web/guides/primary-components/navigation-or-route-changers) section, there is a component called a `NavLink` that I think would be handy for this situation given it has a way to declare an `activeClassName`, which can be used to handler the bolding or any other styles you'd like to add.

This is all leading us to greatly simplify and really _focus_ the `Home`, `ListView` and `AddView` components. These components can then just return some placeholder text like `<h2>Home</h2>` and so on.

Because I'm probably not great at explaining myself, here's an example of what I'm talking about. (Intentionally not giving away all the code details 😜 )

<img src="https://user-images.githubusercontent.com/8881164/125949002-c7957681-ee1f-4dee-9797-b36e18f7ef58.gif" width="450px" />

Reply:
This is a great suggestion @kevinfalting. I saw this comment after writing my review about the `<NavLink>`. 

Let's try to shorten this code and avoid repetitive render. I'm also happy to discuss it in more detail :)